beacon-chain/blockchain/receive_block.go

```diff
@@ -468,6 +469,9 @@ func (s *Service) validateStateTransition(ctx context.Context, preState state.Be
 	stateTransitionStartTime := time.Now()
 	postState, err := transition.ExecuteStateTransition(ctx, preState, signed)
 	if err != nil {
+		if electra.IsExecutionRequestError(err) {
+			return nil, err
+		}
```

Comment:
I wouldn't check this here, this leaks internal logic up the stack, instead, if you keep the error in `ProcessOperations`, you can check on the caller.